packages/appium/lib/logsink.js

```diff
@@ -8,7 +8,7 @@ logger.patchLogger(npmlog);
 global._global_npmlog = npmlog;
 
 // npmlog is used only for emitting, we use winston for output
```

Comment:
is the comment still relevant?

Reply:
it's... mostly still relevant.  it's true when the server is running.